docs/arguments.md

```diff
@@ -92,6 +92,67 @@ The ingester query API was improved over time, but defaults to the old behaviour
 - `-distributor.extra-query-delay`
    This is used by a component with an embedded distributor (Querier and Ruler) to control how long to wait until sending more than the minimum amount of queries needed for a successful response.
 
+- `distributor.accept-ha-samples`
+   Per-user flag to enable handling of samples with external labels for identifying replicas in an HA Prometheus setup. This defaults to false, and is technically defined in the Distributor limits.
+
+- `distributor.enable-ha-tracker` 
+   Enable the distributors HA tracker so that it can accept samples from Prometheus HA replicas gracefully (requires labels). Global (for distributors), this ensures that the necessary internal data structures for the HA handling are created. The option `accept-ha-samples` is still needed to enable ingestion of HA samples per user ID.
+
+### Ring/HA Tracker Store
+
+The KVStore client is used by both the Ring and HA Tracker.
+- `{ring,ha-tracker}.prefix`
+   The prefix for the keys in the store. Should end with a /. For example with a preifx of foo, the key bar would be stored under foo/bar.
```

Comment:
s/preifx/prefix, also, should probably say "prefix of foo/" since the prefix should end with a /.